EIPS/eip-1123.md

```diff
@@ -1397,8 +1393,6 @@ The `name` field defines which compiler was used in compilation.
 </table>
 
 
-<div id="version-version-1"></div>
```

Comment:
Kramdown generates proper links out the headers now, no need for these. If left here htmlproofer complains about duplicate IDs. Manually verified they point to the right header.